packages/codemirror/test/mode.spec.ts

```diff
@@ -1,28 +1,31 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
-/// <reference types="../typings/codemirror/codemirror" />
+
+// TODO CM6:
```

Comment:
TODO we should provide and test adding a new mode.

Reply:
This can be done in a dedicated PR.